.github/workflows/ci.yml

```diff
@@ -71,14 +71,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Log python version info (${{ matrix.python-version }})
         run: python --version --version
-      - name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
       - name: Pip cache
         uses: actions/cache@v3
         with:
-          path: ${{ steps.pip-cache.outputs.dir }}
+          path: $(pip cache dir)
```

Comment:
You can't do shell invocations right in the action inputs. It must be in a run-step, setting a var that can then be rendered. GHA only interpolates own syntax, not bash.

Reply:
You're supposed not to see this ;D

Reply:
_/me sees everything_